SUMMARY.md

```diff
@@ -12,6 +12,7 @@
   - [Roles](roles.md)
   - [Working Open](working-open.md)
   - [Finance 🚧](finance.md)
+  - [Novel Coronavirus (2019) 🦠](coronavirus.md) 
```

Comment:
What's the emoji, I cannot see it.

Reply:
![Screenshot_2020-04-15_09-25-26](https://user-images.githubusercontent.com/4692834/79342410-173d3600-7efb-11ea-9291-6612bdff389f.png)

Reply:
Should I cut? It's around in most places! 
> "Microbe was approved as part of Unicode 11.0 in 2018 and added to Emoji 11.0 in 2018. "

Reply:
Hah!